This patch is part of a series of patches for festival for
Debian GNU/Linux, to provide Debian specific building for festival

Index: b/config/systems/default.mak
===================================================================
--- a/config/systems/default.mak
+++ b/config/systems/default.mak
@@ -142,3 +142,6 @@
 JAVA_COMPILER_VERSION_COMMAND=true
 
 JAVA_SYSTEM_INCLUDES  = -I$(JAVA_HOME)/include/genunix
+
+GCC_MAKE_SHARED_LIB = gcc -shared -o XXX
+
